app/views/parties/current.html.haml

```diff
@@ -0,0 +1,5 @@
+%h1= t('.title')
+%ul.parties-list
+  - @parties.each do |party|
+    %li
+      = party.respond_to?(:partyName) ? party.partyName : ''
```

Comment:
this has been done. Please see new pull request for branch: aebirim/new-changes-party-controller